### PR TITLE
Cy 1537 add snapshot tests 5.0.5

### PR DESCRIPTION
**Pull request by glukhman**
DO NOT MERGE! not ready yet

### CODE REVIEW

File: cosmo_tester/framework/test_hosts.py [hunk 1]
```diff
@@ -885,11 +886,20 @@ class Cloudify4_6Manager(_OldStatusFormat, _CloudifyManager):
...
+    branch_name = '5.0.5'
+
+
 class Cloudify4_6Manager_ClusterJoined(Cloudify4_6Manager):
     def verify_services_are_running(self):
         pass
 
 
+class Cloudify5_0_5Manager_ClusterJoined(Cloudify5_0_5Manager):
+    def verify_services_are_running(self):
```
Comment (tyacbovi) [inline cosmo_tester/framework/test_hosts.py]: Why is this needed?

Comment (glukhman) [reply cosmo_tester/framework/test_hosts.py]: Not sure. I figured out that it's needed for cluster to bypass the fact that the manager is not an AIO and doesn't have all services locally, but maybe I'm wrong about it. The framework is not very transparent
